src/state/actions/resourceActions.ts

```diff
@@ -8,6 +8,9 @@ export const GET_IDS_SUCCESS = "GET_IDS_SUCCESS";
 export const GET_IDS_FAILURE = "GET_IDS_FAILURE";
 export const GET_STRUCTUREDEF_SUCCESS = "GET_STRUCTUREDEF_SUCCESS";
 export const GET_STRUCTUREDEF_FAILURE = "GET_STRUCTUREDEF_FAILURE";
+
+export const UPDATE_STRUCTUREDEFEXTENSION = "UPDATE_STRUCTUREDEFEXTENSION";
```

Comment:
Same here. I think naming your const like this makes it easier to read

```suggestion
export const UPDATE_STRUCTURE_DEF_EXTENSION = "UPDATE_STRUCTUREDEFEXTENSION";
```

Reply:
I totally agree !